tw-pnl.py

```diff
@@ -819,24 +821,24 @@ def check(all_wk, output_summary, output_csv, output_excel, tax_output, show, ve
         if isnan(quantity):
             quantity = 1
         else:
-            if tcode == 'Receive Deliver' and tsubcode in ('Forward Split' or tsubcode == 'Reverse Split'):
```

Comment:
I assume that this was a bug and not intentional

Reply:
Nice spot. Yes, this was a bogus change in b25fd150